src/handler.rs

```diff
@@ -148,7 +155,32 @@ pub enum Command {
 
 #[derive(Debug, Clone, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct SubstrateCommand {}
+pub enum SubstrateCommand {
+    MixerRelayTx(MixerRelayTransaction),
+}
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MixerRelayTransaction {
+    /// one of the supported chains of this relayer
+    pub chain: String,
+    /// The tree id of the mixer's underlying tree
+    pub id: u32,
+    /// The zero-knowledge proof bytes
+    pub proof: Vec<u8>,
+    /// The target merkle root for the proof
+    pub root: Element,
+    /// The nullifier_hash for the proof
+    pub nullifier_hash: Element,
+    /// The receipient of the transaction
+    pub recipient: AccountData,
+    /// The relayer of the transaction
+    pub relayer: AccountData,
```

Comment:
Same as `recipient` above:
```suggestion
    pub relayer: subxt::sp_core::crypto::AccountId32,
```